web/index.html

```diff
@@ -8,6 +8,6 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="module" src="/src/main.tsx"></script>
+    <script type="module" src="/src/App.tsx"></script>
```

Comment:
여기 
`<script type="module" src="/src/App.tsx"></script>`
이렇게 하니까 라우팅 자체가 작동 안하는데?

원래처럼
```suggestion
<script type="module" src="/src/main.tsx"></script>
```
이렇게 작성하니까 라우팅 동작은 하는데,
<img width="779" alt="image" src="https://github.com/user-attachments/assets/251a4ba9-4780-4d67-83c6-5e7571504796" />
css가 작동하지 않아

Reply:
### 이렇게 떠야 정상!
![image](https://github.com/user-attachments/assets/93a8b2b2-6f82-423e-8e7c-6ce1b84da3fa)
- `main.tsx` 파일이 삭제되고 다시 만드는 과정에서 `index.html` 파일도 함께 좀 꼬인 것 같습니다.
- `main.tsx` 수정 완료
- `index.tsx` 수정 완료 (아래 내용 참고)
- `src="/src/main.tsx"` 수정
- `<link href="src/index.css" rel="stylesheet" />` 추가

Reply:
오 잘 나온다
<img width="1194" alt="image" src="https://github.com/user-attachments/assets/f52ab11b-6f04-4aeb-9efb-d737657974ee" />

그럼 이제 코드 리뷰 들어갈게 @elfffffy